package.json

```diff
@@ -26,10 +26,11 @@
     "lint-fix": "yarn prettier --write **/*.js **/*.ts ./programs/**/*.rs ./contracts**/*.sol && cargo +nightly fmt  --all && cargo clippy",
     "clean-fast": "for dir in node_modules cache cache-zk artifacts artifacts-zk dist typechain; do mv \"${dir}\" \"_${dir}\"; rm -rf \"_${dir}\" &; done",
     "clean": "rm -rf node_modules cache cache-zk artifacts artifacts-zk dist typechain",
+    "generate-svm-assets": "sh ./scripts/generate-svm-assets.sh",
```

Comment:
we should think about making this file easier to reason about with this new command you've added. the equivalent evm command is `generate-contract-types`. I tried with the other commands to make it clear what they are (solidity/js/rust/evm/svm) but this one is now a bit inconsistent. perhaps if we call it `generate-svm-assets` and then re-name the `generate-contract-types` to `generate-evm-assets`?

also, I think that given this command is not yet within the `prepublish` command we could actually be safe to merge this into master without it joining the released package.

Reply:
Updated ✅

> also, I think that given this command is not yet within the prepublish command we could actually be safe to
merge this into master without it joining the released package.

I think it is, its part of the build as we need to run `yarn generate-svm-assets` before `yarn build-ts`. See this:
https://github.com/across-protocol/contracts/blob/907b7d7982a90ee456bff87a9cef179f51b23d87/package.json#L33

Let's wait for @mrice32 & @nicholaspai to see what they think the proposed changes.